### PR TITLE
Fixes issue #1402 Updated join the team section

### PR DESCRIPTION
**Pull request by nj4710**
Fixes #1421

### CODE REVIEW

**Comment by nj4710** [comment]
r ?@zoepage

**Comment by miketaylr** [comment]
Ping @zoepage :)

File: webcompat/static/css/development/components/threesteps.css [hunk 1]
```diff
@@ -25,8 +25,18 @@
...
       /* img */
-      .wc-Threesteps-item-number img {
-        width: 50px;
+      
+      .wc-bug-report-icon {
+        width:120px;
+
+      }
+      .wc-bug-diagnosis-icon {
+        width: 100px;
```
Comment (zoepage) [inline webcompat/static/css/development/components/threesteps.css]: please change to `width: 85px;`

File: webcompat/static/css/development/components/threesteps.css [hunk 2]
```diff
@@ -25,8 +25,18 @@
...
+        margin-top: 10px;
+      }
+      .wc-site-outreach-icon {
+        width:100px;
```
Comment (zoepage) [inline webcompat/static/css/development/components/threesteps.css]: please change to `width: 85px;`

File: webcompat/static/css/development/components/threesteps.css [hunk 3]
```diff
@@ -25,8 +25,18 @@
...
+        margin-top: 10px;
```
Comment (zoepage) [inline webcompat/static/css/development/components/threesteps.css]: please change to `margin-top: 5px;`

File: webcompat/static/css/development/components/threesteps.css [hunk 4]
```diff
@@ -36,6 +46,12 @@
       }
     }
 
+    @media all and (max-width: 59.37em) {
+      .wc-bug-report-icon {
+        width: 100px;
+      }
```
Comment (zoepage) [inline webcompat/static/css/development/components/threesteps.css]: Please add 
```
.wc-site-outreach-icon,
 .wc-bug-diagnosis-icon {
    width: 70px;
}
```

**Comment by zoepage** [comment]
Hey @nj4710, if you resolve the merge conflicts, we should be good to merge :)

**Comment by nj4710** [comment]
I am not sure but did it get resolved? @zoepage

**Comment by zoepage** [comment]
@nj4710 YES!
One last tiny detail!

Could you run `git commit --amed` and change the commit message to `Issue #1402 - add new content to join the team section ` or something similar? And were good! :tada:

**Comment by nj4710** [comment]
Which commits should be squashed? This was the squashed commit.  

> 	Fixes issue #1402 Updated join the team section			db54350

The first one happened by mistake when I was resolving conflicts. 
Are you talking about squashing the merge and last commit?

**Comment by zoepage** [comment]
@neha No worries. :)
As you can see here your commit history https://github.com/nj4710/webcompat.com/commits/newbranch you did put the commits together already. Well done! 👍 

But the wording of the commit is the last thing we need to fix. :)
You can do this by `git commit --amend`. The commit message will pop up in your terminal. Now you just need to change the wording to `Issue #1402 - Updated join the team section` and `git push -f origin newbranch`. This should fix the problem :)

More info: https://de.atlassian.com/git/tutorials/rewriting-history

**Comment by zoepage** [comment]
LGTM! r+

Thank you for your patience @nj4710 :)

**Comment by nj4710** [comment]
Thanks to you @zoepage  :)

**Comment by Neha** [comment]
Guys,

I think you are tagging wrong Neha here :)

**Pull request merged**